azure_pipelines/templates/tests-with-conf-file.yml

```diff
@@ -87,4 +84,11 @@ steps:
       targetPath: '$(Agent.BuildDirectory)/s/test_output/'
       artifactName: 'TestOutputs Attempt - $(System.StageAttempt) - ${{ parameters.schema }}'
       publishLocation: 'pipeline'
+  - task: PublishPipelineArtifact@1
+    condition: failed()
+    displayName: "Publish crash report if any"
+    inputs:
+      targetPath: "/Users/runner/Library/Logs/DiagnosticReports"
```

Comment:
Does such a folder exist in the pipeline? I think you'd need to create it, if it doesn't

Reply:
Hi Ameya, yes, it is there always (or maybe there is always certain system crash?). It is a macOS system owned folder to store crash logs from macOS